data/download_data.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+from pprint import pprint
+import logging
+
+logger = logging.getLogger(__name__)
+
+class download_metadata():
+
+    def __init__(
+        self,
+        api_url: str) -> None:
+        self.api_url = api_url
+        self.response_api = requests.get(self.api_url)
+        while True:
+            if self.response_api == 200:
+                logger.info(f"The api resposne is successful")
+            else:
+                logger.warning(f"The api resposne is unsuccessul, please insert correct url")
+                break
+    
+    def get_data(self)-> None:
+             
+        # Get the data from the resposne
+        self.raw_data = self.response_api.text
+
+        # Parse the data into json format
+        data_json = json.loads(self.raw_data)
+        data_first_record = data_json['records'][0]
+        pprint(data_first_record)
```

Comment:
When making this a function, maybe add a arguments for printing, and not print by default.

Reply:
sure, I'll add a conditional argument if someone needed to print and see the data!